CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+![fork](https://user-images.githubusercontent.com/66238394/114072909-590b7d00-98c0-11eb-90dd-fe4b63c9238f.png)
+
+- Open a pull request by clicking the Create pull request button.
+
+![fork](https://user-images.githubusercontent.com/66238394/114072909-590b7d00-98c0-11eb-90dd-fe4b63c9238f.png)
+
+- Then add a title and description to your pull request that explains your contribution. Click on _Create Pull Request_ button.
+
+Voila! Your Pull Request has been submitted and will be reviewed by the moderators and merged. :partying_face:
+
+## Creating an issue
+
+Issues can be used to keep track of bugs, enhancements, or other requests.
+
+- On GitHub, navigate to the main page of the repository.
+- Under your repository name, click on the Issues button.
+
+![issue](https://user-images.githubusercontent.com/66238394/114073960-7ee55180-98c1-11eb-98ad-77954d5f76db.png)
+
+- Click New issue.
+
+![newissue](https://user-images.githubusercontent.com/66238394/114075186-e64fd100-98c2-11eb-98b7-8d8cf6c18eba.png)
+
+- If there are multiple issue types, click Get started next to the type of issue you'd like to open.
+
+![issuetype](https://user-images.githubusercontent.com/66238394/114075732-80177e00-98c3-11eb-8bd1-e75b08521915.png)
+
+- Type a title and description for your issue.
+- After creating the issue you have to wait until the project maintainer assigns the issue to you.
+
+## Add a new article :page_with_curl:
+
+- Install the Project locally. You need to use NodeJS and Yarn (Instructions are on [README.md](README.md))
+- Find the directory in `docs` where this should ideally go. Let's say it goes in searching-algorithms, so create a file in there named interpolation-search.mdx
+- Once you do that, go over to sidebar.js in your forked repo, and go over to this [line](https://github.com/TesseractCoding/NeoAlgo-Docs/blob/781088e0adbcebf313213e937ebe5ed21564bbd2/sidebars.js#L17) and add searching-algorithms/interpolation-search
+- The basic layout for the article should be:
+  - Introduction
+  - Explanation
+  - Algorithmic Steps
+  - Code (Use C++/Java and Python; At least two languages should be present and pick up code from NeoAlgo with apt credits to author)
+  - Sample I/O
+  - Time & Space Complexity
+- Always check the application build and see if your article is being correctly rendered on the local server
+
+## Add a blog post :writing_hand:
+
+- Create a file at `blog/2021-02-28-greetings.md`:
+
+```md title="blog/2021-02-28-greetings.md"
+---
+title: Greetings!
+author: Author Name
+author_title: NeoAlgo Docs Contributor
+author_url: https://github.com/<author_username>
+author_image_url: https://github.com/<author_username>.png
+---
+
+Here goes your blog content
+```
+
+A new blog post is now available at `http://localhost:3000/blog/greetings`.
+
+
+## Summary
+
+- [Contribution workflow](#Step-1-:-Find-an-issue)
+- [How to create an article?](#add-a-new-article-:page_with_curl:)
+- [How to create a blog?](#add-a-blog-post-:writing_hand:)
```

Comment:
Are the hyperlinks working?

Reply:
yes

> Are the hyperlinks working?

yes!

Reply:
They don't seem to. Atleast on my side.

![image](https://user-images.githubusercontent.com/47351025/114150395-a1b34c80-9939-11eb-9fa1-96e8058931ce.png)

Reply:
It's working on VSCode. This is supposed to work on GIthub as well. Please have a look and if the problem persists, suggest me a fix.